src/shims/x86/mod.rs

```diff
@@ -468,6 +468,86 @@ fn unary_op_ps<'tcx>(
     Ok(())
 }
 
+enum ShiftOp {
+    /// Shift left logic
+    Sll,
+    /// Shift right logic (shift in zeros)
+    Srl,
+    /// Shift right arithmetic (shift in sign)
+    Sra,
```

Comment:
```suggestion
    /// Shift left, logically (shift in zeros) -- same as shift left, arithmetically
    Sll,
    /// Shift right, logically (shift in zeros)
    Srl,
    /// Shift right, arithmetically (shift in sign)
    Sra,
```
Would it make sense to rename `Sll` to `Sl` since there is only one kind of left-shift?

Reply:
What about?

```rust
enum ShiftOp {
    Left,
    RightLogic,
    RightArith,
}
```

Since the S is actually redundant

Reply:
Sounds good :)